rpc/wrtc_base_channel.go

```diff
@@ -206,12 +206,13 @@ func (ch *webrtcBaseChannel) onChannelClose() {
 // abort chunk that is not indicative of an actual state of error.
 func isUserInitiatedAbortChunkErr(err error) bool {
 	return err != nil && errors.Is(err, sctp.ErrChunk) &&
-		strings.Contains(err.Error(), "User Initiated Abort: Close called")
+		strings.Contains(err.Error(), "User Initiated Abort:")
 }
 
 func (ch *webrtcBaseChannel) onChannelError(err error) {
 	if errors.Is(err, sctp.ErrResetPacketInStateNotExist) ||
 		isUserInitiatedAbortChunkErr(err) {
+		ch.onChannelClose()
```

Comment:
@dgottlieb this plus a few lines above ends up fixing a case where some tests don't see the channel close with an error "in time"